scripts/release-test-script-platform.sh

```diff
@@ -0,0 +1,341 @@
+#!/bin/bash
```

Comment:
this is ut-control release testing script.

Reply:
## ut-control build testing script

|#| TARGET | Docker | Expectation |
|-|--------|--------|-------------|
|1| build TARGET=linux| None|library build|
|2| build TARGET=arm| rdk-dunfell| library build|
|3| build TARGET=arm| rdk-kirkstone| library build|
|4| build TARGET=arm| comcast-vm| library build |
|5| build TARGET=linux| None|buid test|
|6| build TARGET=arm| rdk-dunfell| build test |
|7| build TARGET=arm| rdk-kirkstone| build test |
|8| build TARGET=arm| comcast-vm| build test |